README.md

```diff
@@ -1,89 +1,88 @@
-- [CodeIgniter](#codeigniter)
-- [Cpp](#cpp)
-- [Crystal](#crystal)
-- [CSS](#css)
-- [D](#d)
-- [Dart](#dart)
-- [Deep Learning](#deep-learning)
-- [Design Patterns](#design-patterns)
-- [Django](#django)
-- [Docker](#docker)
-- [Electron](#electron)
-- [Flutter](#flutter)
-- [Flux](#flux)
-- [Git](#git)
-- [GNU/Linux](#gnulinux)
-- [Go](#go)
-- [Gömülü Sistemler](#gömülü-sistemler)
-- [GraphQL](#graphql)
-- [Gulp](#gulp)
-- [Haskell](#haskell)
-- [HTML](#html)
-- [Ionic](#ionic)
-- [Java](#java)
-- [JavaScript](#javascript)
-- [jQuery](#jquery)
-- [Kotlin](#kotlin)
-- [Laravel](#laravel)
-- [Machine Learning](#machine-learning)
-- [MATLAB](#matlab)
-- [MongoDB](#mongodb)
-- [Nginx](#nginx)
-- [Node.js](#nodejs)
-- [OpenSSL](#openssl)
-- [Packer](#packer)
-- [PHP](#php)
-- [Postgres](#postgres)
-- [PowerShell](#powershell)
-- [Python](#python)
-- [Rails](#rails)
-- [React](#react)
-- [React Native](#react-native)
-- [Ruby](#ruby)
-- [Rust](#rust)
-- [RxJs](#rxjs)
-- [Sketch](#sketch)
-- [Solidity](#solidity)
-- [Sourcemod](#sourcemod)
-- [Spring Boot](#spring-boot)
-- [SQL](#sql)
-- [Swift](#swift)
-- [Titanium Mobile](#titanium-mobile)
-- [TypeScript](#typescript)
-- [Vault](#vault)
-- [Vim](#vim)
-- [Vue.js](#vuejs)
-- [WordPress](#wordpress)
-- [Xamarin](#xamarin)
-- Diğer Kaynaklar
+### Konu Başlıkları
+  - [Android](#android)
+  - [Angular](#angular)
+  - [ASP.NET](#aspnet)
+  - [Arduino](#arduino)
+  - [Bash](#bash)
+  - [Bilgisayar Bilimleri](#bilgisayar-bilimleri)
+  - [Blockchain](#blockchain)
+  - [Bot Framework](#bot-framework)
+  - [C](#c)
+  - [Csharp](#csharp)
+  - [Clojure](#clojure)
+  - [CodeIgniter](#codeigniter)
+  - [Cpp](#cpp)
+  - [Crystal](#crystal)
+  - [CSS](#css)
+  - [D](#d)
+  - [Dart](#dart)
+  - [Deep Learning](#deep-learning)
+  - [Design Patterns](#design-patterns)
+  - [Django](#django)
+  - [Docker](#docker)
+  - [Electron](#electron)
+  - [Flux](#flux)
+  - [Flutter](#flutter)
+  - [Git](#git)
+  - [GNU/Linux](#gnulinux)
+  - [Go](#go)
+  - [Gömülü Sistemler](#gömülü-sistemler)
+  - [GraphQL](#graphql)
+  - [Gulp](#gulp)
+  - [Haskell](#haskell)
+  - [HTML](#html)
+  - [Ionic](#ionic)
+  - [Java](#java)
+  - [JavaScript](#javascript)
+  - [jQuery](#jquery)
+  - [Kotlin](#kotlin)
+  - [Laravel](#laravel)
+  - [Machine Learning](#machine-learning)
+  - [Matlab](#matlab)
+  - [MongoDB](#mongodb)
+  - [Nginx](#nginx)
+  - [Node.js](#nodejs)
+  - [OpenSSL](#openssl)
+  - [Packer](#packer)
+  - [PHP](#php)
+  - [Postgres](#postgres)
+  - [PowerShell](#powershell)
+  - [Python](#python)
+  - [Rails](#rails)
+  - [React](#react)
+  - [React Native](#react-native)
+  - [Ruby](#ruby)
+  - [Rust](#rust)
+  - [RXJS](#rxjs)
+  - [Sketch](#sketch)
+  - [Solidity](#solidity)
+  - [Sourcemod](#sourcemod)
+  - [Spring Boot](#spring-boot)
+  - [SQL](#sql)
+  - [Swift](#swift)
+  - [Titanium Mobile](#titanium-mobile)
+  - [TypeScript](#typescript)
+  - [Vault](#vault)
+  - [Veri Bilimi (Data Science)](#veri-bilimi-data-science)
```

Comment:
New section? Please move into its own PR.

Reply:
There was a distinction in the link section below for this. But there was no go link in the category list. Actually, it was not newly added.

Reply:
👍